### PR TITLE
Reverts the health changes made to jobs.

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request
Alternative to https://github.com/sojourn-13/sojourn-station/pull/3563

I don't understand these changes. Why were they made? I don't really see a reason for our jobs to be even more combat-specialized than before. 

This PR can be merged on a whim, but I encourage some discussion on it in the comments. 

## Changelog
:cl:
balance: Jobs now all have 100 HP
/:cl:

### CODE REVIEW

File: code/game/jobs/job/captain.dm [hunk 1]
```diff
@@ -17,7 +17,6 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 	playtimerequired = 2500
 	wage = WAGE_COMMAND
 
-	health_modifier = 1
```
Comment (Trilbyspaceclone) [inline code/game/jobs/job/captain.dm:20]: colonist+ should have 1 more hp imo

Comment (nikothedude) [reply code/game/jobs/job/captain.dm:20]: honestly i really dont see why

Comment (nikothedude) [reply code/game/jobs/job/captain.dm:20]: the premier already has /things/, why give them a /singular/ hp

Comment (Trilbyspaceclone) [reply code/game/jobs/job/captain.dm:20]: Its colonist+ they should have something to denote that they are better then the rest

Comment (nikothedude) [reply code/game/jobs/job/captain.dm:20]: ඞ

**Comment by nikothedude** [comment]
ALright at trillby's request I'll put some raesoning as to why this shouldn't stay

giving jobs intrinsic HP is just a bad idea. If we want to rebalance the combat performance of a job, we should change their equipment or stats, not their HP. Science being too powerful in combat can be changed through forcing them to behave differently, by nerfing their equipment, or by limiting their potential by lowering their ANA. 

Lowering HP is only a nerf in the beginning of the round, before they have enough ana and insights to raise their HP above others. 

Plus, this change affects more than just science or medical or whatever-it buffs blackshield, it buffs marshals, it buffs prospectors, while nerfing most others. Why does medical need to be made of paper mache? Are they a problem? 

Also, what about non-combat damage? Things like reagents, things like shock, etc. That should also be considered with a change like this. 

And finally-this is just a really unpopular change. I know for some reason we don't really take this into account, but a lot of people seem to really just /not/ like this change. I am one of them.

**Comment by nikothedude** [comment]
ok dont make ana negative worst mistake of my life

**Comment by cdb-is-not-good** [comment]
As always with the weekends, i'll be rather busy but give me some time and i'll sit down and look this over.

**Comment by nikothedude** [comment]
Im enforcing a break from this server on myself, so for the forseeable future I won't engage with this PR, so feel free to close it, as I'm not going to respond to anything said here until I get over myself

**Comment by cdb-is-not-good** [comment]
I'm not against merging this, though the reason I originally stood with adding it was to allow for a buff to some jobs so that I could begin the process of getting rid of BRUTE_MOD, flat damage reduction which has caused some small issue for balance

**Comment by cdb-is-not-good** [comment]
If we removed both it'd result in a similar scenario where suddenly these jobs are nerfed for little reason

**Comment by cdb-is-not-good** [comment]
it was...always...craftable by everyone, it's an Eris holdover.

**Comment by nikothedude** [comment]
OH i can close this now

**Pull request closed without merging**